lib/shared/dialogs.dart

```diff
@@ -40,6 +42,11 @@ Future<T?> showThunderDialog<T>({
                 ),
         ),
         actions: [
+          if (tertiaryButtonText != null)

```

Comment:
Could we make this tertiary action be on the left-hand side of the dialog? This might help with accidental taps on this action!

Reply:
Sure! Let me know if this looks good.

| ![image](https://github.com/thunder-app/thunder/assets/7417301/e379687b-a917-4326-8327-970327a7f19e) |
| - |

Reply:
Looks good to me!

Reply:
Just a heads up, it looks like dialogs without a tertiary button now have the buttons left-aligned, so I will take a look at that when I can!